docs/data/charts/funnel/FunnelStacked.js

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import { FunnelChart } from '@mui/x-charts-pro/FunnelChart';
+
+export default function FunnelStacked() {
+  return (
+    <Box sx={{ width: '100%', maxWidth: 500 }}>
+      <FunnelChart
+        sx={{ '.MuiFunnelSection-series-big': { filter: 'brightness(0.7)' } }}
+        series={[
+          {
+            id: 'big',
+            data: dataBig,
+            sectionLabel: {
+              position: { horizontal: 'end' },
+              textAnchor: 'start',
+              margin: { right: -10 },
```

Comment:
it's weird to anchor the text on start and set the right margin. Looks like their is a edge case I don't get 🤔

Reply:
margin was `inside-out`. I've changed the behaviour in the current offset so it slightly more intuitive. But pretty much positive values go up/right, negative down/left